api/repositories/app_repository_test.go

```diff
@@ -135,16 +130,15 @@ var _ = Describe("AppRepository", func() {
 		)
 
 		BeforeEach(func() {
-			cfApp1 = createApp(space1.Name)
-			querySpaceName = space1.Name
+			querySpaceName = space.Name
 		})
 
 		AfterEach(func() {
-			Expect(k8sClient.Delete(context.Background(), cfApp1)).To(Succeed())
+			Expect(k8sClient.Delete(context.Background(), cfApp)).To(Succeed())
```

Comment:
Is this AfterEach needed if we are going to delete the space anyway?

Reply:
Deleting spaces in env tests does not delete the resources they contain. However, as per my previous comment, cleanup in env tests is not really needed.